app.py

```diff
@@ -41,6 +41,7 @@ def assets(path):
 SEARCH_TOP_K = os.environ.get("SEARCH_TOP_K", 5)
 SEARCH_STRICTNESS = os.environ.get("SEARCH_STRICTNESS", 3)
 SEARCH_ENABLE_IN_DOMAIN = os.environ.get("SEARCH_ENABLE_IN_DOMAIN", "true")
+AZURE_AUTH_TYPE = os.environ.get("AZURE_AUTH_TYPE", "rbac")
```

Comment:
Can we actually have this default to "keys" and set it to "rbac" from the bicep deployment? The reason to have it default to keys is that for the case of deployment from the Azure OpenAI Studio chat playground, role assignments are not configured by default between the resources, so keys need to be used by default in that scenario.

Reply:
👍 Good point! I've just pushed that change.